antora.yml

```diff
@@ -7,4 +7,4 @@ nav:
 asciidoc:
   attributes:
     page-origin-private: false
-    neo4j-version: '5'
+    neo4j-version: '2025.01'
```

Comment:
Can we just set this to `2025`, to avoid having to update it every month? @recrwplay

Reply:
I don't think there is such a thing as 'Neo4j version 2025' in the same way that there is for 'Neo4j version 5'